documentation/modules/creating-migration-quick-plan-existing-source.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="creating-migration-quick-plan-existing-source_{context}"]
+= Creating and running a migration plan based on a source provider
+
+Once you have a source provider and a destination provider, you can create and run a migration plan.
+
+.Procedure
+
+. In the {ocp} web console, click *Plans for virtualization* and then click *Create Plan*.
++
+The *Select source provider* page opens.
+. Select the source provider of the VMs you want to migrate and then click *Next*.
++
+The *Select virtual machines* page opens.
```

Comment:
you already selected the vms in the previous step of choosing the source provider and vms for the migration

Reply:
@yaacov Remember the user selects the source, then clicks Next. Only after the user clicks Next do they see the VMs. That's why they are separate steps -- the user must click Next to go from one part of the interface to the other.

Reply:
> Only after the user clicks Next do they see the VMs.

That is not how it is in 2.6.0, in 2.6.0 you select "what you want to migrate" in one step.

A user select the source provider, then the source vms in the same form, no "next click" required, the "next" button is disabled at this point, when select source provider+vms step is done, a user will click the "next" button to continue to the next step of "to where you want to migrate" and select the target provider+namespace+mappings.

We did discuses (in yesterday ad-hoc mtg) separating the "select source provider (and vms)" into "select source provider" and a separate "select source vms", I commented separating the step is technically chalanging, but it can be done, not for 2.6.0.

Reply:
![image](https://github.com/kubev2v/forklift-documentation/assets/60698649/a188a182-450b-42e1-b901-07ca3d6698fd)

Reply:
Click Create Plan and you see this:
![image](https://github.com/kubev2v/forklift-documentation/assets/60698649/cdc44c4a-833b-4465-b19d-dfd2a965432d)

All you can see is the Providers. You cannot see the VMs.

Reply:
Click the provider -- you're correct -- you do not click Next.
![image](https://github.com/kubev2v/forklift-documentation/assets/60698649/22ac9f87-74b4-4b87-a678-2f0e5e8fd364)
Now you see the VMs

Reply:
So, we're both right and wrong. 

It should be 
. In the {ocp} web console, click *Plans for virtualization* and then click *Create Plan*.
+
The  *Create migration plan* wizard opens to the *Select source provider* interface.
. Select the source provider of the VMs you want to migrate.
The *Select virtual machines* interface opens.
. Select the VMs you want to migrate and click *Next*.

That's from the user perspective. It's all one page from the dev perspective. 

Did this change from yesterday? I thought you had to click Next to go from Selecting source provider to selecting virtual machines.

Reply:
> Did this change from yesterday (maybe a new build?) ? 

it was built like that from the start, this is why it's technically chalanging to separate, and I think we will only be able to separate the "select provider" from the "select vm" in 2.7 and not in 2.6.1

Reply:
@yaacov Then I've been mistaken from the start. My apologies. There's no need to change the wizard, as far as I'm concerned. Does my adding an "extra" step make sense to you ? I just want the users to know that what they see changes when they click the source provider.

Reply:
> Does my adding an "extra" step make sense to you ? 

Describing "select vms" as a different step then "select provider" is ok from my side
My concern is with the "click `Next`" when it's not necessary, and the step indicator on the left:

a. we should be clear that we don't need to click "next" between this tow stepa
b. we also need to be aligned with the step indicator on the left, currently it show "step 1" for both "select provider" and for "select vms" so we need a way to align here too

Reply:
@ahadas Please review this as QA.